src/components/command-bar/commands/search/components/documentation-tab-contents/documentation-hit.tsx

```diff
@@ -0,0 +1,22 @@
+import { productSlugsToNames } from 'lib/products'
+import { CommandBarLinkListItem } from 'components/command-bar/components'
+import { DocumentationHitObject, DocumentationHitProps } from './types'
+
+const DocumentationHit = ({ hit }: DocumentationHitProps) => {
+	const { objectID, _highlightResult, product } = hit
+	const { page_title, description } = _highlightResult
+	const productName = product === 'hcp' ? 'HCP' : productSlugsToNames[product]
+
+	return (
+		<CommandBarLinkListItem
+			key={objectID}
```

Comment:
suggestion: As these aren't being rendered as part of a map, this `key` prop doesn't have any practical usage. I see you are providing a key when rendering `<DocumentationHit>`, so we're covered there!

Reply:
Resolved in: https://github.com/hashicorp/dev-portal/pull/990/commits/8c8f40caded817e6bda10e2b52beb1c94dac3085